en/Obsidian Sync/Select files and settings to sync.md

```diff
@@ -1,4 +1,21 @@
-Any files or settings that have been synced to your [[Local and remote vaults|remote vault]] count towards your [[Sync limitations#How large can each remote vault be|storage space]]. By default, [[Introduction to Obsidian Sync|Obsidian Sync]] does not sync larger files such as audio, video, and PDFs.
+Any files or settings synced to your [[Local and remote vaults|remote vault]] contribute to your [[Sync limitations#How large can each remote vault be|storage space]]. By default, [[Introduction to Obsidian Sync|Obsidian Sync]] activates **selective sync** for the following file types:
+- Images
+- Audio
+- Videos
+- PDFs
+
+To sync additional file types, manually toggle `Sync all other types`.
+
+The default **vault configuration sync** settings include:
+- Other file types
+- Main settings
+- Appearance
+- Themes and snippets
+- Hotkeys
+- Active core plugin list
+- Core plugin settings
+
+To sync community plugins, manually toggle `Active community plugin list` and `Installed community plugin list`.
```

Comment:
I just looked and I think we always use bold for setting names, not code.

Reply:
ope, good catch.